docs/cloud/cloud-troubleshooting/issue-creating-database.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Creating database issues
+layout: default
+parent: Cloud troubleshooting
+grand_parent: Cloud
+nav_order: 5
+---
+
+# Creating database issues.
+
+## Issue - Your service has been stopped.
+
+You will be unable to create new databases
+
+### Cause
+
+Please contact us at se@featurebase.com for the reason of service being stopped.
+
+### Solution
+
+Please contact us at [se@featurebase.com](mailto:se@featurebase.com) immediately to prevent loss of data and restore your sevice.
+
+## Issue - You have exhausted your trial credit balance.
```

Comment:
You need a separate page for each An issue/cause/solution to make them easier to find.
Refer to other issues for guidance on the YAML page title (used in nav)
The first heading should be the specific issue being experienced